modules/ROOT/pages/enabling-einstein.adoc

```diff
@@ -0,0 +1,113 @@
+= Enabling Einstein for Anypoint Platform
+
+When you establish a tenant relationship with a Salesforce organization that has Einstein enabled, you can enable Einstein for your Anypoint Platform organization and store your Einstein generative AI audit and feedback data in Salesforce Data Cloud. You can then use that data for compliance, analytics, prompt adjustments, and LLM tuning to enhance your generative AI applications.
+
+As an organization administrator, you can assign permission to the users in your organization so they can use Einstein with Anypoint Platform products that have Einstein capabilities, including:
+
+* Anypoint Code Builder
+* Intelligent Document Processing (IDP)
+
+
+== Before You Begin
+
+Before enabling Einstein in your Anypoint Platform organization, you must:
+ 
+* Have administrative access to an Anypoint Platform organization that isn't a trial organization and isn't expired
+* Have the Organization Administrator permission and role
+* Have administrative access to a Salesforce organization that https://help.salesforce.com/s/articleView?id=sf.generative_ai_enable.htm&type=5[has Einstein enabled]. 
+* xref:access-management::trusted-salesforce-org.adoc[Establish a tenant relationship] between your Anypoint Platform organization and a trusted Salesforce organization. 
+
+[[enable-einstein-salesforce]]
+== Enable Einstein in the Salesforce Organization
+
+When the Salesforce administrator enables Einstein for the connected Anypoint Platform organization, Einstein requests are routed through the connected Salesforce organization. Billing and tracking are done in the Salesforce organization. 
```

Comment:
Is there a better verb for "are done"? The Salesforce organization handles billing and tracking? The Salesforce organization tracks requests and handles billing?

Should tracking come before billing?